<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MetricsAggregateWidget</class>
 <widget class="QWidget" name="MetricsAggregateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Metrics aggregates</string>
  </property>
  <layout class="QGridLayout" name="widgetGridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="minLabel">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minimum:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="minimumLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="maxLabel">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Maximum:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="maximumLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="avgLabel">
     <property name="font">
      <font>
       <family>Open Sans</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Average:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="averageLineEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>minimumLineEdit</tabstop>
  <tabstop>maximumLineEdit</tabstop>
  <tabstop>averageLineEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
